pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -1284,6 +1285,19 @@ public void addSchema(Schema schema, boolean override, boolean force)
     }
   }
 
+  public void updateSegmentsZKTimeInterval(TableConfig tableConfig, Schema schema) {
+    String schemaName = schema.getSchemaName();
+    LOGGER.info("Updating segment zookeeper time interval metadata: {}", schemaName);
```

Comment:
Log the table name instead of the schema name
```suggestion
    LOGGER.info("Updating segment time interval in ZK metadata for table: {}", tableConfig.getTableName());
```